tests/performance/results.js

```diff
@@ -9,7 +9,9 @@ const { median, getResultsFilename } = require( './utils' );
 
 const testSuites = [
     'home-classic-theme',
+    'home-classic-theme-l10n',
     'home-block-theme',
+    'home-block-theme-l10n',
 ];
```

Comment:
Why are the `admin` tests not included here?

Reply:
Because we have too many places where the test suite names are hardcoded, ugh. Fixing now.